app/jobs/fastly_log_processor.rb

```diff
@@ -0,0 +1,75 @@
+require 'zlib'
+FastlyLogProcessor = Struct.new(:bucket, :key) do
```

Comment:
I dont like Structs. they are usually slower than normal classes too. Can we define a class here instead?